pkg/pxc/app/deployment/binlog-collector.go

```diff
@@ -128,12 +128,21 @@ func GetBinlogCollectorDeployment(cr *api.PerconaXtraDBCluster) (appsv1.Deployme
 
 func getStorageEnvs(cr *api.PerconaXtraDBCluster) ([]corev1.EnvVar, error) {
 	storage := cr.Spec.Backup.Storages[cr.Spec.Backup.PITR.StorageName]
+	verifyTLS := "true"
+	if storage.VerifyTLS != nil && !*storage.VerifyTLS {
+		verifyTLS = "false"
+	}
+
 	switch storage.Type {
 	case api.BackupStorageS3:
 		if storage.S3 == nil {
 			return nil, errors.New("s3 storage is not specified")
 		}
 		envs := []corev1.EnvVar{
+			{
+				Name:  "VERIFY_TLS",
```

Comment:
@pooknull are you sure that we will not have a restart of binlogcollector during the update from 1.13.0 to 1.14.0? I think we need to check CR version. Also, we need to add a test.

Reply:
https://github.com/percona/percona-xtradb-cluster-operator/pull/1397/commits/b7e6a571620836572f77db4bc88505967d915068, https://github.com/percona/percona-xtradb-cluster-operator/pull/1397/commits/d66636b05bbc7a8e5c9bd9d42f2f9d41808879b3